definitions/npm/express_v4.x.x/flow_v0.32.x-/express_v4.x.x.js

```diff
@@ -89,6 +94,7 @@ declare class express$Response extends http$ServerResponse mixins express$Reques
   status(statusCode: number): this;
   type(type: string): this;
   vary(field: string): this;
+  req: express$Request;
```

Comment:
It'll be interesting to see if we get feedback on this one when we PR it back to flow-types.  Express definitely uses it internally, but I wonder if it's something they don't want to expose publicly.  Let's keep it and see what happens 😄